### PR TITLE
formatted flags in aerospike_benchmark.py

### PR DESCRIPTION
**Pull request by ishita2411**

### CODE REVIEW

File: perfkitbenchmarker/linux_benchmarks/aerospike_benchmark.py [hunk 1]
```diff
@@ -204,20 +204,20 @@ def GetConfig(user_config):
           FLAGS.gce_ssd_interface = FLAGS.server_gce_ssd_interface
         config['vm_groups']['clients']['vm_spec']['GCP']['num_local_ssds'] = 0
 
-  if FLAGS.aerospike_server_machine_type:
+  if AEROSPIKE_SERVER_MACHINE_TYPE:
```
Comment (pmkc) [inline perfkitbenchmarker/linux_benchmarks/aerospike_benchmark.py:207]: I think this needs to be AEROSPIKE_SERVER_MACHINE_TYPE.value

File: perfkitbenchmarker/linux_benchmarks/aerospike_benchmark.py [hunk 2]
```diff
@@ -204,20 +204,20 @@ def GetConfig(user_config):
...
     for cloud in vm_spec:
-      vm_spec[cloud]['machine_type'] = FLAGS.aerospike_client_machine_type
+      vm_spec[cloud]['machine_type'] = AEROSPIKE_CLIENT_MACHINE_TYPE
 
   if FLAGS['aerospike_vms'].present:
     config['vm_groups']['workers']['vm_count'] = FLAGS.aerospike_vms
 
-  if FLAGS['aerospike_client_vms'].present:
-    config['vm_groups']['clients']['vm_count'] = FLAGS.aerospike_client_vms
+  if AEROSPIKE_CLIENT_VMS.present:
```
Comment (pmkc) [inline perfkitbenchmarker/linux_benchmarks/aerospike_benchmark.py:219]: I don't think .present works with flag holders so maybe leave it as it was.

**Review by pmkc — CHANGES_REQUESTED** [review]

**Comment by pmkc** [comment]
Hi Ishita sorry we got cut off:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork is what I was going to suggest for pulling changes to your repo (and then pulling from your own GitHub)

File: perfkitbenchmarker/linux_benchmarks/aerospike_benchmark.py (hunk 2 above)
Comment (jellyfishcake) [reply perfkitbenchmarker/linux_benchmarks/aerospike_benchmark.py]: Either leave it as it was or do:
if AEROSPIKE_CLIENT_VMS.value:
...

**Review by pmkc — APPROVED** [review]

**Pull request merged**